shared/src/main/java/uk/gov/di/authentication/shared/services/AuthSessionService.java

```diff
@@ -40,7 +40,7 @@ public AuthSessionService(
         this.configurationService = configurationService;
     }
 
-    public void addOrUpdateSessionId(
+    public void updateSessionIncludingSessionId(
```

Comment:
I think perhaps `addOrUpdateSessionIncludingSessionId` might be better here. In the case that an `oldItem` can't be found, it will create a new one, with the log message 
`"New item added to Auth session table"`
which I don't think is expected from a method which says it updates a session.

This is definitely a tricky one to name though. I think this method might be doing too much to be honest. But I think `addOrUpdateSessionIncludingSessionId` does capture all that it does. 

What do you reckon?

Reply:
Yep sure I can go with that 👍